css/style.css

```diff
@@ -264,6 +273,10 @@ button.button-inline:hover {
 	overflow: hidden;
 	display: flex;
 	min-height: 40px;
+	position: fixed;
```

Comment:
Why fixed position? It just creates problems:

1. ![image](https://user-images.githubusercontent.com/4129927/27387056-e75c12aa-5697-11e7-9611-c45cba0433e0.png)

2. you have to give "imaginary" `margin-top` to `.card-list` (line 303). Everything seems to work just as fine without those 3 declarations
3. mobile issue mentioned by @artemanufrij 

It works just fine if you remove those 3 lines.